packages/integration-karma/scripts/shared/options.js

```diff
@@ -14,16 +14,16 @@ if (process.env.NATIVE_SHADOW) {
 
 const COMPAT = Boolean(process.env.COMPAT);
 const DISABLE_SYNTHETIC = Boolean(process.env.DISABLE_SYNTHETIC);
-const FORCED_NATIVE_SHADOW_MODE_FOR_TEST = Boolean(process.env.FORCED_NATIVE_SHADOW_MODE_FOR_TEST);
+const FORCE_NATIVE_SHADOW_MODE_FOR_TEST = Boolean(process.env.FORCE_NATIVE_SHADOW_MODE_FOR_TEST);
 const TAGS = [`${DISABLE_SYNTHETIC ? 'native' : 'synthetic'}-shadow`, COMPAT && 'compat'].filter(
-    (v) => Boolean(v)
+    Boolean
 );
 
 module.exports = {
     // Test configuration
     COMPAT,
-    DISABLE_SYNTHETIC,
-    FORCED_NATIVE_SHADOW_MODE_FOR_TEST,
+    FORCE_NATIVE_SHADOW_MODE_FOR_TEST,
+    SYNTHETIC_SHADOW_ENABLED: !DISABLE_SYNTHETIC,
```

Comment:
State of synthetic shadow can hereafter be referenced by `SYNTHETIC_SHADOW_ENABLED` to avoid double negatives in the codebase.